deployment/flatpak/usage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Flatpak Usage
+subsection: flatpak
+section: deployment
+---
+
+# {{page.title}}
+
+## Installing Flatpak Applications
+
+### Setting up the Flathub repository: 
+
+A team of Flatpak developers have started a project known as [Flathub](https://flathub.org). Flathub aims to provide a centralized repository for making Flatpak applications available to users. Flathub covers more than the GNOME application suite, and is regularly adding new applications.
+
+To add the Flathub repository, open the Terminal, and run:
+
+```
+$ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+```
+
+
+
+### Installing Applications (GNOME Software): 
+
+GNOME Software already supports Flatpak repositories, so applications can be installed directly from GNOME Software.
```

Comment:
And user runs f.e. KDE?
What then?

Reply:
Non-GNOME users are recommended to use the Command Line instructions.
I should probably have mentioned that somewhere.
I'll add a sentence about it.

Reply:
Great!